glide/src/main/kotlin/br/com/concrete/yosef/glide/GlideImageComponent.kt

```diff
@@ -24,10 +29,11 @@ class GlideImageComponent(
     }
 
     private val commands: Map<String, DynamicPropertyCommand> = mapOf(
-        WidthCommand.WIDTH_TYPE to WidthCommand(),
-        HeightCommand.HEIGHT_TYPE to HeightCommand(),
+        WIDTH_TYPE to WidthCommand(),
+        HEIGHT_TYPE to HeightCommand(),
+        SCALE_TYPE to ScaleTypePropertyCommand(),
```

Comment:
Very nice :smile:, it's possible to add an adjustViewBounds options? or make it true by default. I had this problem while testing.

Reply:
Sure!

Reply:
I think you missed the ScaleTypePropertyCommand file. Theres a value options in which adjustViewBounds is on! Use scaleAspectFit option